Makefile

```diff
@@ -26,23 +31,26 @@ dist/tool-install: Makefile
 	mkdir -p dist
 	touch dist/tool-install
 
+.PHONY: dist/buf
 dist/buf: dist/buf-go dist/buf-web
 
 dist/buf-go: dist/tool-install $(PROTO_FILES)
-	PATH=$(PATH_WITH_TOOLS) buf lint
-	PATH=$(PATH_WITH_TOOLS) buf format -w
-	PATH=$(PATH_WITH_TOOLS) buf generate --template ./proto/viam/buf.gen.yaml
-	PATH=$(PATH_WITH_TOOLS) buf generate --template ./proto/viam/buf.gen.tagger.yaml
+	buf lint
+	buf format -w
+	buf generate --template ./proto/viam/buf.gen.yaml
+	buf generate --template ./proto/viam/buf.gen.tagger.yaml
 	touch dist/buf-go
 
 dist/buf-web: dist/tool-install $(PROTO_FILES)
-	PATH=$(PATH_WITH_TOOLS) buf lint
-	PATH=$(PATH_WITH_TOOLS) buf format -w
-	PATH=$(PATH_WITH_TOOLS) buf generate --template ./proto/viam/buf.gen.web.yaml
-	PATH=$(PATH_WITH_TOOLS) buf generate --timeout 5m --template ./proto/viam/buf.gen.web.yaml buf.build/googleapis/googleapis
+	buf lint
+	buf format -w
+	buf generate --template ./proto/viam/buf.gen.web.yaml
+	buf generate --timeout 5m --template ./proto/viam/buf.gen.web.yaml buf.build/googleapis/googleapis
+	touch dist/buf-web
 
+.PHONY: lint
 lint: dist/tool-install
-	PATH=$(PATH_WITH_TOOLS) buf lint
-	PATH=$(PATH_WITH_TOOLS) buf format -w
+	buf lint
+	buf format -w
 	export pkgs=`go list -f '{{.Dir}}' ./... | grep -v gen | grep -v proto` && echo "$$pkgs" | xargs go vet -vettool=$(TOOL_BIN)/combined
 	export GOGC=50 pkgs=`go list -f '{{.Dir}}' ./... | grep -v gen | grep -v proto` && echo "$$pkgs" | xargs $(TOOL_BIN)/golangci-lint run $(LINT_BUILD_TAGS) -v --fix --config=./etc/.golangci.yaml
```

Comment:
If you remove TOOL_BIN you can remove it here too. just "combined" and "golangci-lint" should work since it'll be in the PATH now.